### PR TITLE
fix(app): fix protocol dashboard dqa

### PR DESCRIPTION
**Pull request by koji**
# Overview
update protocol dashboard, protocol card, pinned protocol to fix dqa feedback

close RAUT-455

# Test Plan

# Changelog
- update ProtocolDashboard
- update ProtocolCard
- export No protocols part as NoProtocols
- update SmallButton padding (with icon the size of padding 16px)

# Review requests

# Risk assessment
low

### CODE REVIEW

File: app/src/App/OnDeviceDisplayApp.tsx [hunk 1]
```diff
@@ -245,7 +245,7 @@ export const OnDeviceDisplayApp = (): JSX.Element => {
   }, [dispatch, isIdle, usersBrightness])
 
   return (
-    <ApiHostProvider hostname="localhost">
```
Comment (vegano1) [inline app/src/App/OnDeviceDisplayApp.tsx]: leftover from debug?

Comment (koji) [reply app/src/App/OnDeviceDisplayApp.tsx]: no, for odd we use `localhost` since the app and robot-server is on the same machine.

Comment (shlokamin) [reply app/src/App/OnDeviceDisplayApp.tsx]: yeah i think @vegano1 is saying you overwrote with `192.168.0.107`

Comment (vegano1) [reply app/src/App/OnDeviceDisplayApp.tsx]: ya

**Review by shlokamin — APPROVED** [review]
lgtm just revert the change you made overwriting the ip to api host provider

File: app/src/App/OnDeviceDisplayApp.tsx (hunk 1 above)
Comment (koji) [reply app/src/App/OnDeviceDisplayApp.tsx]: oh thanks, @vegano1 !

**Comment by ewagoner** [comment]
A few observations:

- long names aren't truncated and spill into the date
- when there is only one protocol, its card should be full width
- the header is wonky (but my protocol dashboard PR should fix that)

Screenshot 2023-06-13 at 2 52 33 PM
Screenshot 2023-06-13 at 2 56 16 PM

**Comment by koji** [comment]
> A few observations:
> 
> * long names aren't truncated and spill into the date
> * when there is only one protocol, its card should be full width
> * the header is wonky (but my protocol dashboard PR should fix that)
> 
> Screenshot 2023-06-13 at 2 52 33 PM Screenshot 2023-06-13 at 2 56 16 PM

that is weird on my end I cannot reproduce long name issues.

**Comment by ewagoner** [comment]
> that is weird on my end I cannot reproduce long name issues.

It will only happen if you have a name that is one long string of characters with no spaces, like `0309_OpentronsRisingStarsPromotionPartyExtravaganza`.

**Comment by ewagoner** [comment]
> It will only happen if you have a name that is one long string of characters with no spaces

I believe I had the `overflowWrap="anywhere"` for the flex containing the name to force it to break even when there was no whitespace.

**Comment by koji** [comment]
> > It will only happen if you have a name that is one long string of characters with no spaces
> 
> I believe I had the `overflowWrap="anywhere"` for the flex containing the name to force it to break even when there was no whitespace.

thank you @ewagoner.
back overflowWrap="anywhere" to Flexbox.

**Comment by ewagoner** [comment]
Looks great now! I'll give it my blessing :)

Screenshot 2023-06-13 at 4 35 29 PM

**Review by ewagoner — APPROVED** [review]

**Pull request merged**